downstream/modules/hub/proc-create-super-users.adoc

```diff
@@ -4,16 +4,16 @@
 
 = Creating a super user
 
-You can create a super user in automation hub and spread administration work across your team. 
+If you want to spread administration across your team, you can create a super user in {HubName}. 
 
 .Prerequisites
 
-* You have *Super user* permissions and can create users in automation hub.  
+* You must be a *Super user*.  
 
 .Procedure
 . Log in to your local {HubName}.
-. From the navigation panel, select menu:User Access[Users].
-. Select the user you want to be a super user to see the User details page.
-. Select *Super User* under User type.
+. Navigate to menu:User Access[Users].
+. Select the user that you want to make a super user. The *User details* for that user are displayed.
+. Under *User type*, select *Super User* under User type.
```

Comment:
This sentence has some duplication and one instance of "under user type" should be removed.